lib/glific/domain_create.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Glific.DomainCreate do
+  @moduledoc """
+  This module adds domains via Gigalixir API using the Tesla module.
+  """
+
+  @endpoint "https://api.gigalixir.com/api/apps/glific-staging/domains"
+  @username "vgull574@stanford.edu"
+  @api_key "4f91902b-6d0f-4068-9f76-2c7916ff0b79"
```

Comment:
We should not expose the username and api_keys in codebase, instead we should fetch it from env variables.
You can refer 
https://github.com/glific/glific/blob/master/config/runtime.exs
https://github.com/glific/glific/blob/9579bc8d5944fe5ab52fecfe348758e0455524cf/lib/glific/notion.ex#L23 on how to add, and then load env variables into code.

You can also read the different ways we use  and load env variables in elixir here https://hexdocs.pm/elixir/Application.html#module-the-application-environment